src/yang/ietf-bgp-types.yang

```diff
@@ -288,6 +293,10 @@ module ietf-bgp-types {
     // 4-octet value:
     //  <as number> 2 octets
     //  <community value> 2 octets
+    // XXX JMH - Note that experience has shown that restricting ranges of
+    // communities may have negative impact on things such as policy, which may
+    // need to refer to out-of-acceptable-range values.  I suggest not
+    // restricting these beyond limiting the range to UINT32_MAX.
```

Comment:
Marked for discussion.

Reply:
Flagged as an issue that needs to be resolved. As the comment says, the range can be restrictive in certain cases and prevents operators from trying to do the right thing. Should the model allow for an unrestricted range and let things fail, or should it try to define two typedefs, one that is restrictive and one that allows for any range.

Reply:
Created an issue and will track it there.